backend/layers/business/business.py

```diff
@@ -5,12 +5,14 @@
 from datetime import datetime
 from functools import reduce
 from typing import Dict, Iterable, List, Optional, Set, Tuple
+from backend.common.corpora_config import CorporaConfig
 
 from backend.layers.business.business_interface import BusinessLogicInterface
 from backend.layers.business.entities import (
     CollectionMetadataUpdate,
     CollectionQueryFilter,
     DatasetArtifactDownloadData,
+    DatasetArtifactDownloadDataDeprecated,
```

Comment:
Added `Deprecated` suffix to existing `DatasetArtifactDownloadData` specific to pre-signed URL functionality. The `XxDeprecated` entities and methods are to be removed in #5697.